hack/deployer/runner/gke.go

```diff
@@ -167,7 +169,15 @@ func (d *GkeDriver) create() error {
 	log.Println("Creating cluster...")
 	pspOption := ""
 	if d.plan.Psp {
-		pspOption = " --enable-pod-security-policy"
+		pspOption = "--enable-pod-security-policy "
+	}
+
+	cidrOptions := ""
+	if d.plan.Gke.ClusterIPv4CIDR != "" {
+		cidrOptions += fmt.Sprintf("--cluster-ipv4-cidr=%s ", d.plan.Gke.ClusterIPv4CIDR)
+	}
+	if d.plan.Gke.ServicesIPv4CIDR != "" {
+		cidrOptions += fmt.Sprintf("--services-ipv4-cidr=%s ", d.plan.Gke.ServicesIPv4CIDR)
```

Comment:
Nit: `strings.Join` may be nicer and safer to handle spaces in this string concatenation.

```go
baseCmd := "x new --a --b=c"

opts := []string{}
opts = append(opts, "--enable-pod-security-policy")
if true {
	opts = append(opts, "--yy=zz")
}

cmd := fmt.Sprintf("%s %s", baseCmd, strings.Join(opts, " "))
```